resources/js/src/app/shared/interfaces.ts

```diff
@@ -32,9 +32,14 @@ export interface Facet {
 }
 
 export interface FacetValue {
+    count?: string;
+    image?: string;
     id?: string;
     name: string;
     imageUrl?: string;
+    items?: [];
+    position?: string;
```

Comment:
It would be better if we could also make this a `number` similar to `count`.

Reply:
The value is not numeric. All the position values I've seen some far were 'item'

Reply:
Can you check where this comes from and if we even deal with this at all? We probably don't need stuff that we don't even use/depend on.

Reply:
`Findologic\Api\Response\Parser\FiltersParser::parseFilter()`
Looks like it expects the `$key` in `foreach ($filter->items->item as $key => $item) {` to be numeric which it is not.
Can't really see any place where it is being accessed.

Reply:
We do have a wrapper for our search-api, which is called [findologic-api](https://github.com/findologic/findologic-api/).

[There](https://github.com/findologic/findologic-api/blob/develop/src/Responses/Xml21/Xml21Response.php#L71) we only go through all filters without even respecting that key. I do not even think that it makes sense to store this information for views to consume, especially if we don't even use it and it doesn't hold any useful information for us.

```php
        foreach ($xmlResponse->filters->main->children() as $filter) {
            $filterName = ResponseHelper::getStringProperty($filter, 'name');
            // Set filter names as keys for the filters.
            $this->mainFilters[$filterName] = Filter::getInstance($filter);
            $this->hasMainFilters = true;
            $this->mainFilterCount++;
        }
```